app/assets/javascripts/application.js

```diff
@@ -1,6 +1,6 @@
 //= require require_config
 
-require(['common', 'jquery'], function(MAS, $) {
+require(['common', 'jquery', 'wantToTalk'], function(MAS, $) {
```

Comment:
ok, have moved the code into a component module, although it isn't a dough-style component as such. Hope this is ok.